pkg/plugins/optional/grafana/v1alpha/scaffolds/init.go

```diff
@@ -51,5 +51,6 @@ func (s *initScaffolder) Scaffold() error {
 	return scaffold.Execute(
 		&templates.RuntimeManifest{},
 		&templates.ResourcesManifest{},
+		&templates.CustomMetricsConfigManifest{ConfigPath: string(ConfigFilePath)},
```

Comment:
I think we do not need to pass the path because it will be fixed always
So, we can have a const with the default path inside of the template or am I missing something?

Reply:
@camilamacedo86 this path is fixed for now. But I instead on passing it here so that we can make it configurable later. That way, if we want to introduce a feature where the user would like to scaffold config file anywhere else in the project, we can read and accept it. Wdyt?